### PR TITLE
Update text-wrap style. Closes #346

### LINKED ISSUE #346: Enhancement to reduce widows and orphans in the theme
Sometimes text can have single lines of text (widows and orphans) from a paragraph that dangle at either the beginning or end of a block of text, which blocks readability. Unless [this Gutenberg issue](https://github.com/WordPress/twentytwentyfive/issues/25#issuecomment-2315517591) gets implemented, one solution could be adding the code below (or something similar that achieves the same) in `style.css` to prevent that from happening.

image

```
/* Progresive enhancement to reduce widows and orphans.
/* <https://github.com/WordPress/gutenberg/issues/55190>
*/
h1, h2, h3, h4, h5, h6, blockquote, caption, figcaption {
		text-wrap: balance; /* Safari fallback */
		text-wrap: pretty;
}
p {
    text-wrap: pretty;
}

```

### PR DESCRIPTION
**Pull request by bobmatyas**
**Description**

Per #346, this adds the `text-wrap` style to deal with widows and orphans.

**Screenshots**

Before Example:

before-text-wrap

After Example:

after-text-wrap

**Testing Instructions**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -28,3 +28,16 @@ a {
 	outline-width: 2px;
 	outline-style: solid;
 }
+
+/*
+ * Progresive enhancement to reduce widows and orphans
+ * https://github.com/WordPress/gutenberg/issues/55190
+ */
+h1, h2, h3, h4, h5, h6, blockquote, caption, figcaption {
+	text-wrap: balance; /* Safari fallback */
```
Comment (ryelle) [inline style.css]: `balance` can be an issue for CJK languages (and possibly others) ([see this core ticket](https://core.trac.wordpress.org/ticket/60892)). If it's just a fallback, I would suggest removing it and letting the fallback be default wrapping.

Comment (bobmatyas) [reply style.css]: @ryelle Thanks for letting me know. I followed the suggested approach on the GH issue and wasn't aware of the core ticket. I have removed `balance`.

**Review by carolinan — CHANGES_REQUESTED** [review]
Hi, please remove changes that are unrelated to the text-wrap.

**Comment by bobmatyas** [comment]
@carolinan I think I was able to remove the extraneous commits. 🤞

File: style.css [hunk 2]
```diff
@@ -28,3 +28,11 @@ a {
...
 	outline-style: solid;
 }
+
+/*
+ * Progresive enhancement to reduce widows and orphans
+ * https://github.com/WordPress/gutenberg/issues/55190
+ */
+h1, h2, h3, h4, h5, h6, blockquote, caption, figcaption, p {
+	text-wrap: pretty;
+}
```
Comment (carolinan) [inline style.css]:
```suggestion
}

```
There needs to be a single blank line at the end of the document.

**Review by carolinan — APPROVED** [review]

**Comment by carolinan** [comment]
This style is only enqueued on the front, and I think this is the right decision.
Without extensive testing, we won't know if there would be any negative impacts on the interface if the style would be added to the editor. Some parts of the editor already use text-wrap on these elements.

**Pull request merged**